src/test/java/com/mwo/group2/StudentTest.java

```diff
@@ -1,5 +1,87 @@
 package com.mwo.group2;
 
-public class StudentTest extends User {
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.Before;
 
+public class StudentTest extends User {
+	
+	/*
+    @Before

```

Comment:
This was commented because there are 2 ways to initialize "student" in the test part - 1st option - under @Before (this is what I commented out) and 2nd option - to repeat initialization in each test.
I'll uncomment it and remove the repeated line for initialization in each test.